debian/control

```diff
@@ -7,6 +7,8 @@ Build-Depends: debhelper (>= 8.0.0),
  libmono-winforms2.0-cil,
  libmono-cairo2.0-cil,
  geckofx29, unzip, wget,
+ pkg-config,
+ curl,
```

Comment:
Oh, it might not need wget AND curl. It didn't occur to me to remove 'wget', but I could if that's thought to be a good idea.